test/functional/test_framework/messages.py

```diff
@@ -31,7 +31,7 @@
 MY_SUBVERSION = b"/python-mininode-tester:0.0.3/"
 MY_RELAY = 1 # from version 70001 onwards, fRelay should be appended to version messages (BIP37)
 
-MAX_INV_SZ = 50000
+# MAX_INV_SZ = 50000
```

Comment:
remove? (This is no longer used after the removal of the comptool)

Reply:
Done!